docs/static/setup.ps1

```diff
@@ -0,0 +1,193 @@
+# Copyright (c) 2020 Ryan Cross

```

Comment:
Sorry I wasn't clear before, can we stick with the single author file approach please, e.g. 


```suggestion
# Copyright (c) 2020 The ZMK Contributors
```